zmon_worker_monitor/builtins/plugins/history.py

```diff
@@ -4,30 +4,43 @@
 # wrapper for kairosdb to access history data about checks
 
 import logging
+import os
+
 import requests
+import tokens
 
 from zmon_worker_monitor.builtins.plugins.distance_to_history import DistanceWrapper
 
 from zmon_worker_monitor.adapters.ifunctionfactory_plugin import IFunctionFactoryPlugin, propartial
 
+
 logger = logging.getLogger(__name__)
 
 
+# will use OAUTH2_ACCESS_TOKEN_URL environment variable by default
+# will try to read application credentials from CREDENTIALS_DIR
+tokens.configure()
+tokens.manage('uid', ['uid'])
+tokens.start()
+
+ONE_WEEK = 7 * 24 * 60 * 60
```

Comment:
True; will be mentioned in the docs.

mainly related to the constructed query here https://github.com/zalando-zmon/zmon-worker/pull/70/files#diff-39537ca0ecb6596a43d3742eec839130R90